### DISCUSSION TITLE
Loading custom Script [Iubenda Example]

### DISCUSSION THREAD

**Discussion by bitsmyth** [vercel/next.js#12223] [Help] [answered]
I want to load the cookie banner solution from iubenda to block all unwanted cookie-scripts. Gatsby has done a plugin for that.
https://www.gatsbyjs.org/packages/gatsby-plugin-iubenda-cookie-footer/

```import Document, { DocumentContext, Head, Main, NextScript, Html } from 'next/document';

const iubendaImplementation = `
<script type="text/javascript">
var _iub = _iub || [];
_iub.csConfiguration = {"countryDetection":true,"perPurposeConsent":true,"whitelabel":false,"lang":"de","siteId":222222,"consentOnDocument":true,"consentOnHorizontalScroll":true,"cookiePolicyId":222222, "banner":{ "acceptButtonDisplay":true,"customizeButtonDisplay":true,"acceptButtonColor":"#65a6a0","acceptButtonCaptionColor":"white","customizeButtonColor":"#212121","customizeButtonCaptionColor":"white","position":"float-top-center","textColor":"white","backgroundColor":"#000001" }};
</script>
<script type="text/javascript" src="//cdn.iubenda.com/cs/iubenda_cs.js" charset="UTF-8" async></script>`;

export default class MyDocument extends Document {
    render() {
        return (
            <Html lang="de-at">
                <Head>
                    <div dangerouslySetInnerHTML={{ __html: iubendaImplementation }}> </div>
                </Head>
                <body>
                    <Main />
                    <NextScript />
                </body>
            </Html>
        );
    }
}
```

With this code I am able to see the cookie banner but I also got an error: 

> index.js:1 Warning: next-head-count is missing. https://err.sh/next.js/next-head-count-missing

**Comment by roelvan** [comment, marked as answer]
You should put this line `<div dangerouslySetInnerHTML={{ __html: iubendaImplementation }}> </div>` under the `<NextScript />` line.

**Comment by jk-baseer** [comment]
@n1xn Hey Man, Do you mind to share exact steps on how you got this working? 

Am getting errors like . This is the similar errors am getting for other script based installations as well. 
![image](https://user-images.githubusercontent.com/61427976/155591387-31839adc-70e5-4816-8728-2fa1fbcdfc26.png)

**Reply by bitsmyth** [reply]
Sorry I don’t know what I was doing back then but I am currently solving all third party script implementations via Google tag manager.

**Comment by dasrajanbvi** [comment]
Hello @roelvan i tried the same like you mentioned but it still doesn't work for me. 

please find below code sample.

`       <NextScript>
                        <div dangerouslySetInnerHTML={{ __html: iubendaImplementation }}> </div>
                    </NextScript>`

**Reply by imarthurg** [reply]
Hey, @dasrajanbvi. It works for me putting the div tag below the <NextScript /> tag, not inner it. Just like:

`<NextScript /><div dangerouslySetInnerHTML={{ __html: iubendaImplementation }} />`
